types/react-native/index.d.ts

```diff
@@ -9594,6 +9594,48 @@ export namespace addons {
     export type TestModule = TestModuleStatic;
 }
 
+/**
+ * Below types are not properly typed in react-native
+ * https://github.com/facebook/react-native/blob/6c563a507fd8c41e04a1e62e2ba87993c6eb1e2f/Libraries/ReactNative/FabricUIManager.js#L21
+ * TODO: Update types according to proper implementation
+ **/
+type FabricNode = object;
+type FabricNodeSet = Array<FabricNode>;
+type FabricNodeProps = object;
+type FabricInstanceHandle = object;
+
+interface NativeFabricUIManagerSpec {
```

Comment:
Not sure that this should be added to the types.  I dont think this is part of the public interface. No-one should be calling these APIs directly.

Reply:
You are right it might be bad idea to share types of private parts. 
What do you think about defining it as an object type?